src/vercheck.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient } from '@actions/http-client'
+
+export interface Vercheck {
+  Version?: string
+  Summary?: string
+  Link?: string
+  Advisory?: string
+}
+
+export async function vercheck(cur: string): Promise<Vercheck> {
+  const c = new HttpClient('atlas-action')
+  const resp = await c.get(`https://vercheck.ariga.io/atlas-action/${cur}`)
```

Comment:
are you adding test=1 in case it's a local testing run?

Reply:
i'm using `nock` to mock the backend